src/components/Error/Error.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Error.scss';
+
+type ErrorProps = {
+  hasError: string;
+  onModalClose: () => void;
+};
+
+export const Error: React.FC<ErrorProps> = ({ hasError, onModalClose }) => (
+  <div className="Error notification is-danger is-light">
+    <div className="Error-message">
+      {`Error: ${hasError}`}
+    </div>
+
+    <button
+      aria-label="modal-close"
+      type="button"
+      className="button is-danger"
+      data-cy="modal-close"
+      onClick={() => onModalClose()}
```

Comment:
You can use a shorter form by just passing reference to the function. You can use it every time when you get the same params that needed to this nested function.
```suggestion
      onClick={onModalClose}
```